app/views/lists/show.html.erb

```diff
@@ -5,6 +5,7 @@
     <ul>
       <% @list.items.each do |item| %>
         <li>
+          <span><%= item.category ? item.category.name : "unknown" %></span>
```

Comment:
small thing, better to use single quotes rather than double quotes, possibly a rails thing but seems to be the style that is used most consistently

Reply:
Unless interpolating then double quotes are required

Reply:
Thanks! Is that just for return values? Should HTML classes etc stay double-quoted?

Reply:
Just for string literals I think but the other guys here will know more from the rails pov